lib/rmt/downloader.rb

```diff
@@ -117,7 +114,7 @@ def make_request(remote_file, local_file, request_fiber, checksum_type = nil, ch
 
     FileUtils.mv(downloaded_file.path, local_file)
 
-    @logger.info("D #{File.basename(local_file)}")
+    @logger.info("↓ #{File.basename(local_file)}")
```

Comment:
Same as for `×` ... might look nice but not sure if we should invent our own symbol language.